examples/heatmap_example.py

```diff
@@ -0,0 +1,63 @@
+"""The script that produces bristlecone_heatmap_example.png."""
```

Comment:
This is not just the bristlecone example anymore.

Reply:
I think we should just have a separate example for interaction map.

Reply:
I will separate this file into two.